Manifests/ManifestsApple/com.apple.system-extension-policy.plist

```diff
@@ -261,6 +261,29 @@
 			<key>pfm_type</key>
 			<string>dictionary</string>
 		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>List of system extension bundle IDs. Extension on the list can be removed by their own app without requiring an admin password.</string>
```

Comment:
I'd like to tweak the wording slightly:
`List of system extension bundle IDs that can be removed by their own app without requiring an admin password.`

Reply:
Sure. It appears that this key was changed last minute so when updating I combined your wording into the new description.